packages/services/f-braze-adapter/src/services/DispatcherEventStream.js

```diff
@@ -0,0 +1,26 @@
+
+class DispatcherEventStream {
+    constructor () {
+        this.subscribers = {};
+    }
+
+    publish (eventName, data) {
+        if (!Array.isArray(this.subscribers[eventName])) {
+            throw new Error(`No event subscribers exist for the following event: ${eventName}`);
```

Comment:
This shouldn't be an error as the client shoudn't know/care whether there are any subscribers.

Reply:
removed as suggested